bcda/cclf/fileprocessor.go

```diff
@@ -56,14 +56,25 @@ func (p *processor) walk(path string, info os.FileInfo, err error) error {
 		return nil
 	}
 
-	zipReader, err := zip.OpenReader(filepath.Clean(path))
+	zipFile := filepath.Clean(path)
+	zipReader, err := zip.OpenReader(zipFile)
+
 	if err != nil {
-		p.skipped = p.skipped + 1
-		msg := fmt.Sprintf("Skipping %s: file could not be opened as a CCLF archive. %s", path, err.Error())
+		modTime, err2 := getModTime(zipFile)
+		if err2 == nil && stillDownloading(modTime) {
+			p.skipped = p.skipped + 1
```

Comment:
I think we wanted the inverse, is that right @laurenkrugen-navapbc? 

- If the file is still downloading, don't update `skipped` and we'll have an alert for any skipped files.
- If the file is not still downloading, increase `skipped` so it triggers the alert.

Reply:
Semantically maybe corrupted files would be better described as `p.corrupted` 🙃

Reply:
@kyeah Sorry, I was focused on my other PR and didn't notice that I hadn't pushed up my logic for the failure counter that @laurenkrugen-navapbc had suggested during our engineering time. Is it alright that I have that? Or should I replace the skipped counter with `corrupted` ? 

And @laurenkrugen-navapbc I removed this error all-together because I realized that getModTime is not necessary since file info is a param in this function.